client/cypress/support/parameters.js

```diff
@@ -0,0 +1,26 @@
+export function dragParam(paramName, offsetLeft, offsetTop) {
+  cy.getByTestId(`DragHandle-${paramName}`)
+    .trigger("mouseover")
+    .trigger("mousedown");
+
+  cy.get(".parameter-dragged .drag-handle")
+    .trigger("mousemove", offsetLeft, offsetTop, { force: true })
+    .trigger("mouseup", { force: true });
+}
+
+export function assertParameterPairSwapping(param1, param2, paramWidth) {
+  cy.server();
+  cy.route("POST", `**/api/*/*`).as("Save");
```

Comment:
This feels a bit risky as any api POST will trigger it 😕. This specific function currently makes sense only for the Dashboard, so maybe we can leave it there for now? (we can just have the `Save` route from the `beforeEach`).

Anyway, let me know what you think in case you have a different idea on how to handle this.